.golangci.yml

```diff
@@ -1,14 +1,16 @@
 run:
     deadline: 2m
-
     skip-dirs:
-        - client
+      - client
+    go: '1.19'
+    build-tags:
+      - e2e
 
 linters:
   disable-all: true
   enable:
     - bodyclose
-    - depguard
+    # - depguard        # NOTE: we want to allow all packages
     - dogsled
     # - dupl            # NOTE: intentionally disabled
```

Comment:
Love these comments, maybe later we should expand on these.

Reply:
Those are Patrik's idea - he gets the credit 😄